fluster/main.py

```diff
@@ -66,7 +66,10 @@ def run(self) -> None:
 
     def _validate_args(self, args: Any) -> None:
         if hasattr(args, "format"):
-            if args.format == SummaryFormat.JUNITXML.value and not args.summary_output:
+            if (
+                args.format in [SummaryFormat.JUNITXML.value, SummaryFormat.CSV.value]
+                and not args.summary_output
+            ):
                 sys.exit(
                     "error: please specify XML file path with -so/--summary-output option."
```

Comment:
```suggestion
                    "error: please specify XML/CSV file path with -so/--summary-output option."
```

Reply:
Thanks for the highlight, I have applied and squashed the change.